src/public/CowSwap.json

```diff
@@ -2,9 +2,9 @@
   "name": "CoW Swap",
   "timestamp": "2022-12-15T19:00:00+00:00",
   "version": {
-    "major": 1,
-    "minor": 0,
-    "patch": 1
+    "major": 0,
```

Comment:
This should bump minor version by one from the current version (https://files.cow.fi/tokens/CowSwap.json), to 1.2.0